specification/cognitiveservices/data-plane/Personalizer/stable/v1.0/Personalizer.json

```diff
@@ -479,6 +479,80 @@
         }
       }
     },
+    "/metrics/learning": {
+      "get": {
+        "tags": [
+          "Metrics"
+        ],
+        "summary": "Get learning performance metrics.",
+        "description": "Get learning performance metrics aggregated by time or events.",
+        "operationId": "LearningMetrics_Get",
+      "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "StartDate",
+            "in": "query",
+            "description": "Start date to get the metrics for.",
+            "required": false,
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "name": "EndDate",
+            "in": "query",
+            "description": "End date to get the metrics for.",
+            "required": false,
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "name": "AggregationInterval",
+            "in": "query",
+            "description": "Aggregation time period to aggregate the results for.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "NumberOfRecentEvents",
+            "in": "query",
+            "description": "Number of recent events.",
+            "required": false,
+            "type": "integer",
+            "format": "int64"
+          },
+          {
+            "name": "AggregationEventCount",
+            "in": "query",
+            "description": "Aggregation event window to aggregate the results for.",
+            "required": false,
+            "type": "integer",
+            "format": "int64"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "array",
```

Comment:
how long can this array become? Could it be come too big to send back to the user? 

Generally good to have a format to allow server driven paging. This way it can be added later if needed without causing a breaking change. There are guidelines here: https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#98-pagination